tests/services-autoraise.nix

```
@@ -0,0 +1,44 @@
+{ config, pkgs, ... }:
+
+let
+  autoraise = pkgs.runCommand "autoraise-0.0.0" { } "mkdir $out";
+in
+
+{
```

We'll also need to set `system.primaryUser` since this is a user-scoped service.
```suggestion
{
  system.primaryUser = "test-autoraise-user";
  
```